lib/job.js

```diff
@@ -7,7 +7,7 @@ var scripts = require('./scripts');
 var debuglog = require('debuglog')('bull');
 var errors = require('./errors');
 
-var FINISHED_WATCHDOG = 5000;
+var FINISHED_WATCHDOG = 1000;
 
```

Comment:
I guess the best would be to have the watchdog timeout as a configurable setting.

Reply:
Making this configurable would be helpful.